notification-support/src/main/java/com/expediagroup/streamplatform/streamregistry/core/events/ConsumerNotificationEventUtils.java

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright (C) 2018-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.streamplatform.streamregistry.core.events;
+
+import static com.expediagroup.streamplatform.streamregistry.core.events.ObjectNodeMapper.serialise;
+import static java.util.Objects.requireNonNull;
+
+import java.util.stream.Collectors;
+
+import lombok.val;
```

Comment:
You're mixing Java 11 `var` and `lombok.val` - please could we converge on only using `var`?

Reply:
checking...

Reply:
I think they are different, `var` is mutable and `val` is inmutable, i.e.:

`val myInt = 9` => `final int myInt = 9`
`var otherInt = 10` => `in otherInt = 10`

Reply:
true but they could all be one or the other. There is no reason to use both in this context.

Reply:
I changed them for `val`. For some reason github it isn't showing the `Outdated` label. But it is ready